packages/Webkul/User/tests/Feature/UserTest.php

```diff
@@ -201,3 +201,204 @@
+    ]);
+
+    $response = $this->put(route('admin.account.update'), [
+        'name'             => 'John',
+        'email'            => 'update@example.com',
+        'current_password' => '',
+        'password'         => '',
+        'image'            => '',
+        'timezone'         => 'Asia/Kolkata',
+        'ui_locale_id'     => 1,
+    ]);
+
+    $response->assertStatus(302);
+    $response->assertSessionHasErrors([
+        'current_password',
+    ]);
+
+    $response->assertInvalid();
+});
+
+it('should not update the user without ui-locale', function () {
+    $this->loginAsAdmin();
+
+    $currentPass = Hash::make('password');
+    $admin = Admin::factory()->create([
+        'email'        => 'update@example.com',
+        'password'     => $currentPass,
+        'ui_locale_id' => 1,
+    ]);
+
+    $response = $this->put(route('admin.account.update'), [
+        'name'             => 'John',
+        'email'            => 'update@example.com',
+        'current_password' => 'password',
+        'password'         => '',
+        'image'            => '',
+        'timezone'         => 'Asia/Kolkata',
+        'ui_locale_id'     => '',
+    ]);
+
+    $response->assertStatus(302);
+    $response->assertSessionHasErrors([
+        'ui_locale_id',
+    ]);
+
+    $response->assertInvalid();
+});
+
+it('should not update the user without timezone', function () {
+    $this->loginAsAdmin();
+
+    $currentPass = Hash::make('password');
+    $admin = Admin::factory()->create([
+        'email'        => 'update@example.com',
+        'password'     => $currentPass,
+        'ui_locale_id' => 1,
+    ]);
+
+    $response = $this->put(route('admin.account.update'), [
+        'name'             => 'John',
+        'email'            => 'update@example.com',
+        'current_password' => 'password',
+        'password'         => '',
+        'image'            => '',
+        'timezone'         => '',
+        'ui_locale_id'     => 1,
+    ]);
+
+    $response->assertStatus(302);
+    $response->assertSessionHasErrors([
+        'timezone',
+    ]);
+
+    $response->assertInvalid();
+});
+
+it('should update the user with all required data', function () {
+    $this->loginAsAdmin();
+
+    $currentPass = Hash::make('password');
+    $admin = Admin::factory()->create([
+        'email'        => 'update@example.com',
+        'password'     => $currentPass,
+        'ui_locale_id' => 1,
+    ]);
+
+    $response = $this->put(route('admin.account.update'), [
+        'name'                  => 'John doe',
+        'email'                 => 'new@example.com',
+        'current_password'      => 'password',
+        'password'              => 'password2',
+        'image'                 => '',
+        'timezone'              => 'Asia/Kolkata',
+        'ui_locale_id'          => 2,
+        'password_confirmation' => 'password2',
+    ]);
+
+    $response->assertStatus(302);
+    $response->assertRedirect();
```

Comment:
Both are same only need to use one assertRedirect checks for the status code 302 also.
```php
    $response->assertRedirect();
```